src/platform/posix/fuzz.c

```diff
@@ -8,10 +8,11 @@
 
 #include <irq_ctrl.h>
 #include <zephyr/sys/time_units.h>
+#include <nsi_cpu_if.h>
 
-/* Zephyr arch APIs, not in a header (native_sim has them though) */
-void posix_init(int argc, char *argv[]);
-void posix_exec_for(uint64_t us);
+/* Zephyr arch APIs, not in a header */
+void nsi_init(int argc, char *argv[]);
```

Comment:
In here they are
https://github.com/zephyrproject-rtos/zephyr/blob/main/scripts/native_simulator/common/src/include/nsi_main_semipublic.h
for this purpose

Reply:
Fixed